manager/orchestrator/services.go

```diff
@@ -230,5 +245,12 @@ func getRunnableSlots(s *store.MemoryStore, serviceID string) (map[uint64]slot,
 		}
 	}
 
-	return runningSlots, nil
+	deadSlots := make(map[uint64]slot)
```

Comment:
@dongluochen: Good point. I've changed the code so that `NEW` is no longer a special case. This was only there for the sake of the drainer unit test. I've fixed the test so it's no longer necessary.